helm/overstroomik/templates/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Can you comment on how to generate these files in a README or similar? I hope that at least some of these templated .yaml files can be generated automatically.

Reply:
helm create